htdocs/install/page_dbsettings.php

```diff
@@ -236,19 +236,19 @@ function select_db($db_name, $link) {
 	if (empty ($error)) {
 		if (!select_db($vars ['DB_NAME'], $link)) {
 			// Database not here: try to create it
-			$result = exec_query("CREATE DATABASE `" . $vars ['DB_NAME'] . '`', $link);
+			$result = exec_query("CREATE DATABASE `" . quote_sql($vars ['DB_NAME']) . '`', $link);
```

Comment:
quote_sql result already starts with quotes, so you need to remove them from string here

Reply:
ok, didn't know it added quotes every time.